webui/src/Emulator/Emulator.tsx

```diff
@@ -65,6 +67,13 @@ export function Emulator() {
 		}
 	}, [retryToken, socket, emulatorId])
 
+	const userConfigStore = useMemo(() => new UserConfigStore(), [])
+	useUserConfigSubscription(socket, userConfigStore)
+
+	useEffect(() => {
+		document.title = `${userConfigStore?.properties?.installName} - Emulator`
+	}, [userConfigStore])
```

Comment:
```suggestion
	}, [userConfigStore.properties?.installName])
```

the `userConfigStore` reference won't ever change after the first render, so this useEffect needs to be more specific and target the property it needs instead.

an alternate option would be to do something more mobx based so that the `useEffect` gets re-run whenever the observable changes, but thats fiddly to do